index.bs

```diff
@@ -915,6 +875,87 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
         </p>
       </section>
 
+      <section>
+        <h4 algorithm>
+          Check a MIME Type
+        </h4>
+        <p>
+          The <dfn id="check-a-mime-type">check a MIME Type</dfn> algorithm takes {{DOMString}} |contentType|, string |media|, and either
+          a {{MediaDecodingType}} or {{MediaEncodingType}} |encodeOrDecodeType|. It
+          returns either failure, or a boolean that represents if the MIME type of |contentType| is [=supported by the user agent=]:
+        </p>
```

Comment:
I propose to use the opportunity to declare the algorithm more appropriately:
- Use an [abstract-op](https://speced.github.io/bikeshed/#dfn-types) dfn type
- Follow the form recommended by the [Infra Standard](https://infra.spec.whatwg.org/#algorithm-declaration) for the declaration of algorithms

Also moving the dfn to the header to make references to it target the section, and moving the `algorithm` attribute to the container to ease automatic extraction.

Note: for some reason Bikeshed is case-sensitive for `abstract-op` references.

```suggestion
      <section algorithm>
        <h4 id="check-mime-type">
          <dfn abstract-op noexport lt="Check a MIME Type|check a MIME Type">
            Check a MIME Type
          </dfn>
        </h4>
        <p>
          To [$check a MIME Type$], given a {{DOMString}} |contentType|, a
          string |media|, and either a {{MediaDecodingType}} or
          {{MediaEncodingType}} |encodeOrDecodeType|, perform the following
          steps. They return either failure, or a boolean that represents if
          the MIME type of |contentType| is [=supported by the user agent=]:
        </p>
```

Reply:
Sounds good to me, thank you!

Reply:
Done in https://github.com/w3c/media-capabilities/pull/222/commits/042414bd06c74ebd1ea8a241e53a15902464a39e. I'll update the other algorithms to follow this in subsequent commits.